src/main/java/org/jitsi/jicofo/bridge/BridgeSelectionStrategy.java

```diff
@@ -86,61 +93,216 @@ public Bridge select(
     }
 
     /**
-     * Selects a bridge to be used for a new participant in a conference,
-     * assuming that no other bridge is used by the conference (i.e. this
-     * is the initial selection of a bridge for the conference).
+     * A1. (Happy case 1): There is a NOL bridge in the conference and in the
+     * region. Use the least loaded of them.
```

Comment:
I think we should update these docs to reflect what the method does, rather than the scenario from the doc.  E.g.:
```
Find a not-overloaded bridge already handling this conference in the participant's region
```